python/whylogs/api/logger/experimental/logger/actor/list_util.py

```diff
@@ -1,17 +1,27 @@
 from itertools import takewhile
-from typing import Any, Generator, List, Tuple, Type
+from typing import Generator, List, Optional, Tuple, Type, TypeVar
 
+K = TypeVar("K")
 
-def get_like_items(items: List[Any]) -> Tuple[List[Any], Type, List[Any]]:
+
+def get_like_items(items: List[K]) -> Tuple[Optional[List[K]], Optional[Type[K]], List[K]]:
```

Comment:
`List[Any]` seems accurate?

Reply:
<verbal discussion>
I don't think it's possible to accurately annotate for both the input and output sides, so annotate how you like and add a comment explaining what's happening.

Reply:
The comments were actually already in the function above. I noted in this function that it just wraps the other one and to see those docs, with a new example in those docs.